ci/scripts/build_kernel_loader.sh

```diff
@@ -3,9 +3,45 @@ set -eux
 source_dir=${1}
 build_dir=${1}/build
 
+rm -rf ${build_dir}
 mkdir -p ${build_dir}
 pushd ${build_dir}
 
+mkdir arrow_tmp
+pushd arrow_tmp
+
+# Download Arrow source code
+url="https://github.com/pitrou/arrow/archive/refs/heads/percentile-rank-kernel.zip"
```

Comment:
To be updated with apache/arrow remote once is merged and with a specific commit or a tag once it's part of a released version.